src/core/graphql/queries/metadata.graphql

```diff
@@ -99,11 +100,7 @@ query priceChart(
   }
 }
 
-query dApp(
-  $shortName: String!
-  $url: String!
-  $status: Boolean!
-) {
+query dApp($shortName: String!, $url: String!, $status: Boolean!) {
```

Comment:
At some point we should figure out what the formatting mismatch is. A lot of files I open now are yellow because you're reformatting with a different config.